src/main/java/com/google/api/codegen/discogapic/transformer/SchemaToViewTransformer.java

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.discogapic.transformer;
+
+import com.google.api.codegen.config.GapicProductConfig;
+import com.google.api.codegen.discovery.Document;
+import com.google.api.codegen.viewmodel.ViewModel;
+import java.util.List;
+
+public interface SchemaToViewTransformer {
```

Comment:
This interface is the same as `DocumentToViewTransformer`, and they serve the same intent, so I don't think we need distinct interfaces.